tests/client-dbus/README.rst

```diff
@@ -0,0 +1,30 @@
+A Library for Testing the stratisd D-Bus layer 
```

Comment:
It's probably Ok to keep this README here, but it would need some editing, since it is no longer for a library.

Reply:
I made the updates